foodstuffs.sk

```diff
@@ -80,17 +84,21 @@ common categories:
 	any fish¦es = any raw fish, any cooked fish
 	
 	any golden apple = normal golden apple, enchanted golden apple
+	any apple = any golden apple, apple
 	[any] (soup|stew)¦s = beetroot soup, mushroom stew, rabbit stew
 	[any] root vegetable¦s = beetroot, carrot, potato
 	any potato¦es = potato, baked potato
-	[any] fruit¦s = apple, chorus fruit, melon slice, any golden apple
+	[any] fruit¦s = any apple, chorus fruit, melon slice
 	
 	[any] raw meat¦s = raw beef, raw chicken, raw mutton, raw porkchop, raw rabbit
 	[any] cooked meat¦s = steak, cooked chicken, cooked mutton, cooked porkchop, cooked rabbit
 	[any] meat¦s = any raw meat, any cooked meat
 	[any] monster (food|meat¦s) = rotten flesh, spider eye
 	[any] cooked food¦s = baked potato, any cooked meat, any cooked fish
 
+	minecraft version = 1.14 or newer

```

Comment:
Have you tested that this actually works? Writing the version specifier most of the way down like this? As I understood it a version tag like this specifies how the entire section parses. Have you tested if any of the aliases in this section load in pre-1.14 versions with this change applied?

Reply:
Oh dang my apologies, I wasn't aware that is how that works. I'll redo it with a section